source/slang/slang-lower-to-ir.cpp

```diff
@@ -2025,6 +2025,14 @@ struct ValLoweringVisitor : ValVisitor<ValLoweringVisitor, LoweredValInfo, Lower
     // type with the appropriate opcode.
     IRType* lowerSimpleIntrinsicType(DeclRefType* type)
     {
+        SLANG_ASSERT(getBuilder()->getInsertLoc().getMode() != IRInsertLoc::Mode::None);
+
+        if (type->declRef.decl && type->declRef.decl->getName()
```

Comment:
What is this all about? type = type seems to indicate it's a nop anyway?

Reply:
Fixed